docs/pages/router/advanced/tabs.mdx

```diff
@@ -3,15 +3,109 @@ title: Tabs
+import { FileTree } from '~/ui/components/FileTree';
+import { Terminal } from '~/ui/components/Snippet';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
 
-Expo Router adds `href` screen option which can only be used with screen options that are an object (for example, `screenOptions={{ href: "/path" }}`) and cannot be used simultaneously with `tabBarButton`.
+## Get started
 
-## Hiding a tab
+### Use a template
 
-Sometimes you want a route to exist but not show up in the tab bar, you can pass `href: null` to disable the button:
+The fastest way to get started with tabs is by using a pre-made template. Run the following command to create a new project with tabs:
+
+<Terminal cmd={['npx create-expo-app --template tabs']} />
+
+### Set up tabs in an existing project
+
+You can use file-based routing to create a tabs layout. Here's an example file structure:
+
+<FileTree
+  files={[
+    'app/_layout.js',
+    'app/(tabs)/_layout.js',
+    'app/(tabs)/index.js',
+    'app/(tabs)/settings.js',
+  ]}
+/>
+
+This file structure will produce a layout with a tab bar at the bottom of the screen. The tab bar will have two tabs: "Home" and "Settings":
+
+<ImageSpotlight
+  alt="A screenshot of a tab bar with two tabs: Home and Settings."
+  src="/static/images/expo-router/tabs.png"
+  style={{ maxWidth: 540 }}
+/>
+
+You can use the **app/\_layout.js** file to define your app's main navigator:
 
 ```jsx app/_layout.js
+import { Stack } from 'expo-router/stack';
+
+export default function Layout() {
+  return (
+    <Stack>
+      <Stack.Screen name="(tabs)" options={{ headerShown: false }}>
+    </Stack>
+  );
+}
+```
+
+The **(tabs)** folder is a special folder name that tells Expo Router to use the `Tabs` layout.
+
+Inside it, the example shows three files. The first is **(tabs)/\_layout.js**. This file is the main layout file for the tab bar and each tab. Inside it, you can control how the tab bar and each time item looks and behaves.
+
+```jsx app/(tabs)/_layout.js
+import React from 'react';
+import FontAwesome from '@expo/vector-icons/FontAwesome';
+import { Tabs } from 'expo-router';
+
+export default function TabLayout() {
+  return (
+    <Tabs screenOptions={{ tabBarActiveTintColor: 'blue' }}>
+      <Tabs.Screen
+        name="index"
+        options={{
+          title: 'Home',
+          tabBarIcon: ({ color }) => <FontAwesome size={28} name="home" color={color} />,
+        }}
+      />
+      <Tabs.Screen
+        name="settings"
+        options={{
+          title: 'Settings',
+          tabBarIcon: ({ color }) => <FontAwesome size={28} name="cog" color={color} />,
+        }}
+      />
+    </Tabs>
+  );
+}
+```
+
+The tabs layout wraps the [Bottom Tabs](https://reactnavigation.org/docs/bottom-tab-navigator) from React Navigation. You can use the options presented on the React Navigation documentation to customize the tab bar.
```

Comment:
```suggestion
The tabs layout wraps the [Bottom Tabs Navigator](https://reactnavigation.org/docs/bottom-tab-navigator) from React Navigation. You can use the options presented in the React Navigation documentation to customize the tab bar.
```

- React Navigation doc specifies it as "Bottom Tab Navigator", let's stick to using the same convention
- Preposition usage is incorrect in the second line.